dojo.py

```diff
@@ -218,3 +219,78 @@ def print_unallocated(self, filename=None):
                 file.write("\n" + "------------------------------------------\n")
                 file.write("{}".format(str1) + "\n")
                 file.close()
+
+    def reallocate_person(self, first_name, last_name, new_room_name):
```

Comment:
This function is really long in my opinion. However, I find that the spacing and naming make it easy enough to follow.